### PR TITLE
Add Telegraf 0.2.0 announcement blog post

### PR DESCRIPTION
**Pull request by gunnaraasen**

### CODE REVIEW

File: content/blog/2015/10/28/announcing_telegraf_0_2_0.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Release Notes
+
+- The `-test` flag will now only output 2 collections for plugins that need it.
+- There is a new agent configuration option: `flush_interval`. This option tells Telegraf how often to flush data to InfluxDB and other output sinks. For example, users can set `interval = "2s"` and `flush_interval = "60s"` for Telegraf to collect data every 2 seconds and flush every 60 seconds.
+- `precision` and `utc` are no longer valid agent config values. `precision` has moved to the InfluxDB output config, where it will continue to default to `s`
+- Debug and test output will now print the raw line-protocol string.
+- Telegraf will now, by default, round the collection interval to the nearest
+even interval. This means that `interval="10s"` will collect every :00, :10, etc.
+- To ease scale concerns, flushing will be "jittered" by a random amount so that
```
Comment (otoolep) [inline content/blog/2015/10/28/announcing_telegraf_0_2_0.md]: I think this needs a little bit more context. Perhaps something like:

- To address scaling concerns Telegraf now adds a small amount of "jitter" to the flush period. This ensures that if a large number of Telegraf instances are deployed, they do not all flush at the same instant.

Comment (gunnaraasen) [inline content/blog/2015/10/28/announcing_telegraf_0_2_0.md]: cc/ @sparrc

@otoolep sounds good to me.

File: content/blog/2015/10/28/announcing_telegraf_0_2_0.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+agent, is now available!
+
+## What’s New
+
+- [PR #205](https://github.com/influxdb/telegraf/pull/205): Include per-db redis keyspace info
+- [PR #241](https://github.com/influxdb/telegraf/pull/241): MQTT Output. thanks [@shirou](https://github.com/shirou)!
+- [PR #240](https://github.com/influxdb/telegraf/pull/240): procstat plugin, thanks [@ranjib](https://github.com/ranjib)!
+- [PR #244](https://github.com/influxdb/telegraf/pull/244): netstat plugin, thanks [@shirou](https://github.com/shirou)!
+- [PR #262](https://github.com/influxdb/telegraf/pull/262): zookeeper plugin, thanks [@jrxFive](https://github.com/jrxFive)!
+- [PR #237](https://github.com/influxdb/telegraf/pull/237): statsd service plugin, thanks [@sparrc](https://github.com/sparrc)
```
Comment (beckettsean) [inline content/blog/2015/10/28/announcing_telegraf_0_2_0.md]: No ! for Cameron?

**Comment by beckettsean** [comment]
LGTM

**Comment by sparrc** [comment]
@gunnaraasen can you make the two changes pointed out by Sean and Philip? I don't think I have write access to your fork

**Comment by sparrc** [comment]
@gunnaraasen I addressed Philip and Sean's comments

**Pull request merged**